ansible/roles/ocp-workload-aiedge-computing-blueprint/tasks/workload.yml

```diff
@@ -381,6 +381,8 @@
   until: "openshift_operators_installplan.resources | json_query('[?status.phase==`RequiresApproval`]')"
   retries: 10
   delay: 30
+  # Continue on if there are no operator installs that require approval
+  ignore_errors: yes
```

Comment:
I'm pretty sure this could be written differently. We also now have a generic `install_operator` role (in the roles directory) that can be used to handle (hopefully) all logic installing and configuring an operator.

See https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/install_operator/tasks/install.yml#L75-L109

Reply:
Thx for pointing this out. I was considering creating something from scratch but this will really help out

Reply:
Updated to use the InstallPlan approver task. Thx for the tip